### PR TITLE
Partial implementation of Roles API

### PR DESCRIPTION
**Pull request by blazejkotowski**
This patch implements most of the actions from Roles API along with
RSpec tests for these actions.

### CODE REVIEW

File: lib/zoom/actions/roles.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  module Actions
+    module Roles
+      def roles_list(*args)
+        options = Params.new(Utils.extract_options!(args))
+        Utils.parse_response self.class.get("/roles", query: options, headers: request_headers)
+      end
+
+      def roles_create(*args)
+        options = Params.new(Utils.extract_options!(args))
+        options.require(%i[name]).permit(%i[description privileges])
```
Comment (kyleboe) [inline lib/zoom/actions/roles.rb:13]: Minor style nit-pick but only use literal syntax when  listing two or more symbols
```suggestion
        options.require(:name).permit(%i[description privileges])
```

File: lib/zoom/actions/roles.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        Utils.parse_response self.class.post("/roles", body: options.to_json, headers: request_headers)
+      end
+
+      def roles_members(*args)
+        options = Params.new(Utils.extract_options!(args))
+        options.require(%i[role_id])
```
Comment (kyleboe) [inline lib/zoom/actions/roles.rb:19]: Same here
```suggestion
        options.require(:role_id)
```

File: lib/zoom/actions/roles.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+      def roles_unassign(*args)
+        options = Params.new(Utils.extract_options!(args))
+        options.require(%i[role_id member_id])
+        Utils.parse_response self.class.delete("/roles/#{options[:role_id]}/members/#{options[:member_id]}", headers: request_headers)
+      end
+
+      def roles_get(*args)
+        options = Params.new(Utils.extract_options!(args))
+        options.require(%i[role_id])
```
Comment (kyleboe) [inline lib/zoom/actions/roles.rb:37]:
```suggestion
        options.require(:role_id)
```

**Comment by blazejkotowski** [comment]
I have committed the requested changes. Thanks to you for your work, it's a pleasure to contribute!

**Pull request merged**